modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase

```diff
@@ -0,0 +1,227 @@
+@component-name = "portal-solutions"
+definition {
+
```

Comment:
Tests are failing on CI.
Could you please try adding skip.portal.instance for these tests and we can run it again to see if that helps.

Reply:
Done